tuf/settings.py

```diff
@@ -80,7 +75,7 @@
 DEFAULT_TARGETS_REQUIRED_LENGTH = 5000000 #bytes
 
 # Set a timeout value in seconds (float) for non-blocking socket operations.
-SOCKET_TIMEOUT = 4 #seconds
+SOCKET_TIMEOUT = 5 #seconds
```

Comment:
Because network connections can be quite lossy in the real world. We found a customer who was timing out over HTTP, but that was more likely due to the stringest `SLOW_START_GRACE_PERIOD` of 0.1 seconds. This latter setting is gone now, because `requests` handles both `connect()` and `read()` timeouts.